src/csharp/Grpc.Core/ContextPropagationToken.cs

```diff
@@ -32,124 +26,10 @@ namespace Grpc.Core
     /// The gRPC native layer provides some other contexts (like tracing context) that
```

Comment:
If `ContextPropagationToken` is not tied to native, is this comment still relevant?

Reply:
Updated the doc comment to stay relevant.